### DISCUSSION TITLE
Losing syntax highlight in long .vue files?

### DISCUSSION THREAD

**Discussion by olivervorasai** [vuejs/language-tools#772] [Q&A] [answered]
When using Volar and viewing very long .vue files the syntax highlighting seems to stop working.

Here is my setup:
1. Using VScode and having only Volar v0.29.8 active, all other extensions disabled.
2. Create a brand new vue project using `vue create hello-world` using latest `vue-cli`
3. Project is Vue 3 + Typescript + Eslint + Prettier
4. This is my vscode settings file:
```
{
    "terminal.integrated.defaultProfile.windows": "Git Bash",
    "editor.tabSize": 4,
    "breadcrumbs.enabled": true,
    "[vue]": {
        "editor.tabSize": 4,
        "editor.defaultFormatter": "johnsoncodehk.volar"
    },
}
```

### Here is what I am seeing at the top of the file:
![image](https://user-images.githubusercontent.com/6283219/145517729-9e38bbc5-6820-4957-a754-8c946c68fd70.png)

### Here is what I am seeing part way down the file:
![image](https://user-images.githubusercontent.com/6283219/145517781-7c24a2c2-9cb4-47ef-a861-591835a0df3d.png)

### The lack of highlighting is even visible in the minimap:
![image](https://user-images.githubusercontent.com/6283219/145518279-6ecd298d-eb47-42d2-8993-0f7c403f1357.png)

I am wondering if there is some dependency or setting I might have missed. I am assuming here that red colors here means volar cannot detect the import component.

Any insight would be appreciated.

**Comment by johnsoncodehk** [comment, marked as answer]
Sorry for missing this issue! This problem is fixed in https://github.com/johnsoncodehk/volar/commit/73ef8103ee04eb901332afb66faf9e8ccb3c13c1.

**Comment by avxkim** [comment]
@johnsoncodehk same issue using this plugin: 
<img width="695" alt="image" src="https://user-images.githubusercontent.com/5739191/207055871-8ad97079-2555-4dc2-a871-779fe14bfd2f.png">

Is it successor of Volar i gues?

**Reply by johnsoncodehk** [reply]
It is Volar, we have renamed the extension in VSCode.
